client/src/extended_bootstrap_css/bootstrap-wet-fixes-extensions.scss

```diff
@@ -273,6 +273,15 @@ div.label {
   border: 2px solid $backgroundColor;
   color: $primaryColor !important;
   border-radius: $standardBorderRadius;
+
+  &--with-icon{/* used for the typeahead filter*/
```

Comment:
Do you think these would be useful elsewhere, or are they too specific to this use case? If they'd be useful elsewhere then you don't need the comment, if they're too tailor-made to the typeahead then move them to local typeahead scss.

`&--modifier` cool! Hadn't seen that use of scss before, handy.